pkg/worker/worker.go

```diff
@@ -294,6 +297,12 @@ func (s *Worker) RunContainer(request *types.ContainerRequest) error {
 	}
 	log.Printf("<%s> - acquired port: %d\n", containerId, bindPort)
 
+	// Setup external S3 mounts
+	err = s.createMountpoints(request)
```

Comment:
Let's move the createmountpoints stuff / mounts stuff into a separate file (worker/mount.go) - you can look at ContainerNetworkManager for an example. I think it might be time to encapsulate all container mount logic (worker side) in a separate struct.